packages/nimble-components/src/table-column/menu-button/cell-view/styles.ts

```diff
@@ -0,0 +1,23 @@
+import { css } from '@microsoft/fast-element';
+import { controlSlimHeight } from '../../../theme-provider/design-tokens';
+
+export const styles = css`
```

Comment:
I was expecting the menu to be the same width as the column. I'm not sure why; maybe because I'm thinking of it like a select? And that would probably not be desirable in all cases, for example if the column is narrow then we wouldn't want the menu to be so narrow that its content is clipped.

I'm curious if we thought about this in the visual design or if we're just getting the default behavior. Assuming it's easy to control the menu width, could you check with Fred/Brandon/the team on what the desired behavior is?

![image](https://github.com/ni/nimble/assets/10500124/1d7f3c34-6c54-41e0-81e6-50ca11e9a58c)

Reply:
This is the default behavior of the menu button itself. I can discuss with Fred and Brandon to see if we're happy with this behavior (both in the column & the stand-alone button).

Reply:
Closing this discussion since the current decision is to keep the menu width as-is.